### ISSUE TITLE
Update Fides CLI to prompt user to register their server

### ISSUE THREAD

**Issue by ThomasLaPiana** [ethyca/fides#1366]

**Comment by seanpreston** [comment]
> @TheAndrewJackson: Should this be done on initial server setup?

@NevilleS to answer.

**Comment by ThomasLaPiana** [comment]
yes, as we can't really prompt during the server setup, we should use the CLI. we can jack into the `fides init` flow for this? or even add an additional check during the `status` check for server registration
